site/layouts/partials/footer.html

```diff
@@ -60,6 +60,7 @@ <h6 class="menu_header" id="menu_4">Learn</h6>
           <h6 class="menu_header" id="menu_5">Connect</h6>
           <ul id="menu_5_drop" class="desktop-show">
             <li><a class="open-intercom-link" href="mailto:support@checklyhq.com">Support</a></li>
+            <li><a href="https://join.slack.com/t/checklycommunity/shared_invite/zt-1gk7uwnht-OEh4SVoTHPpwHHo2QBzbBA">Slack Community</a></li>
```

Comment:
```suggestion
            <li><a href="https://www.checklyhq.com/slack">Slack Community</a></li>
```

Reply:
Unfortunately this link will expire after 100 folks joining. Even though it's one more hoop let's keep the URL in a single place so that we don't have to update multiple places. :)